### DISCUSSION TITLE
Bartycrouch.translate is not working

### DISCUSSION THREAD

**Discussion by mkcpb** [Flinesoft/BartyCrouch#260] [Q&A]
When I call BartyCrouch.translate(key: "some_text", translations: [.english: "Some text"], comment: "Some comment") and build or run the project nothing is happening . Few mouth ago everything works fine. I got L10.someText (for example). Any ideas what could be a problem? 

While I was debugging I found this:

Warning: [BartyCrouch] Untransformed BartyCrouch.
translate(key:translations:comment:) method call found with key 'some_text' and base translations '[CAppCore.BartyCrouch.SupportedLanguage.english: "Some text"]'.
Please ensure that BartyCrouch is installed and configured correctly.

**Comment by Jeehut** [comment]
If this was working before, I assume you already have the build script configured. Can you please check and ensure that the `[update.transform]` section of your `.bartycrouch.toml` file contains the path to the file you have this `BartyCrouch.translate` call in its **`codePaths`**? It sounds like your search paths seem not to match your Swift file path, maybe you renamed things in your project.

Alternatively, the build script might also be broken. For example, if you upgraded to **Xcode 14 beta**, there's a new "Based on dependency analysis" checkbox which you need to uncheck for BartyCrouch to run on each build and therefore work properly. By default it's turned on and looks as below – just uncheck it:
![Screen Shot 2022-09-01 at 10 03 25](https://user-images.githubusercontent.com/6942160/187863964-92351852-dd57-4d8b-9648-78033039ff7b.png)

**Comment by mkcpb** [comment]
Hi, thanks for answer. I tried but it didn't help in my situation. I reinstalled bartycrouch and it's working now.

**Reply by Jeehut** [reply]
@mkcpb That sounds like an easy fix, great to hear. 👍
